Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN curl -L ${NYCDB_REPO}/archive/${NYCDB_REV}.zip > nycdb.zip \
   && pip install .
 
 ARG WOW_REPO=https://github.com/justFixNYC/who-owns-what
-ARG WOW_REV=907eb121a6f41d277152067fe2d792a087bc0ce0
+ARG WOW_REV=2069f8c29f7ce41d116e3c8b42b87ddc0ed69a80
```

Comment:
where does this new value come from?

Reply:
it's just the most recent commit hash on wow, just copy/pasted from here: https://github.com/JustFixNYC/who-owns-what/commits/master

It's a left over kind hacky solution to get the python packages for building the portfolio tables and now the oca tables too. It just downloads the whole repo and builds them in the docker container. I think ideally we would probably just put that elsewhere (maybe just within this repo), or at least just set them up fully as packages we can publish and install here normally.